README.md

```diff
@@ -45,21 +46,20 @@ If you would like to contribute and don't know where to start, or you have any q
 Control*) to clone the repository from https://github.com/acs-upb-mobile/acs-upb-mobile.
 * If the plugins are installed, Android Studio should automatically recognise it as being a Flutter project, so you should be able to just click *Next* on everything and create a new project.
 * When prompted, set up the Flutter SDK (the location is the `flutter/` folder you downloaded when you installed it) and Dart SDK (it comes bundled with Flutter at `flutter/bin/cache/dart-sdk/`).
-* Open the project terminal and switch to the Beta Flutter channel in order to enable web support:
+* Open a terminal in the Flutter SDK directory and switch to the version we are currently using for the project:
 ```
-flutter channel beta
-flutter upgrade
-flutter config --enable-web
+git pull
+git checkout 1.24.0-10.2.pre
 ```
-* Install all of the required packages:
+* Open the project terminal in Android Studio and install all of the required packages by running:
 ```
 flutter pub get
 ```
 * You may need to restart the IDE for the changes to take effect.
 
 ### Building for Android
 
-* Install and run an emulator in Android Studio (using AVD Manager), or connect a physical Android device (make sure USB debugging is enabled).
+* Install and run an emulator in Android Studio (using AVD Manager), or connect a physical Android device (make sure USB debugging is enabled, and your cable supports data transfer).
 * Select your device from the dropdown list and hit the play button (*Shift+F10* or *^R*). Note that Android Studio runs the app in debug mode by default.
 
 | :exclamation: | On Android, ACS UPB Mobile uses **a separate (development) environment in debug mode**. That means a completely different Firebase project - separate data, including user info.|
```

Comment:
\+
_Since the development environment is separate on web, this platform uses  the release version of the app._

Reply:
here, or below _Building for web_

Reply:
Not sure I understand what exactly you mean here, can you clarify?